auth/auth_test.go

```diff
@@ -21,7 +21,6 @@ func (suite *AuthTestSuite) TestMockStore() {
 	suite.Equal(true, Authorize("topics:list_all", []string{"admin", "reader"}, store))
 	suite.Equal(true, Authorize("topics:list_all", []string{"admin", "foo"}, store))
 	suite.Equal(false, Authorize("topics:list_all", []string{"foo"}, store))
-	suite.Equal(false, Authorize("topics:publish", []string{"reader"}, store))
```

Comment:
It's used to demonstrate that role "reader" doesn't have access to the 'publish' action on the resource 'topics'